inference-engine/include/ie_parallel_custom_arena.hpp

```diff
@@ -0,0 +1,157 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+/**
+ * @brief Contains declarations and custom threading interfaces based on TBB info and task_arena APIs.
+ *
+ * @file ie_parallel_custom.hpp
+ */
+
+#pragma once
+
```

Comment:
hope this will not be a part of public interface in a final solution..

Reply:
At this moment it is already moved to the internal part.